db/schema.rb

```diff
@@ -389,21 +390,21 @@
     t.integer  "identifier_scheme_id"
   end
 
-  add_index "user_identifiers", ["user_id"], name: "index_user_identifiers_on_user_id"
+  add_index "user_identifiers", ["user_id"], name: "index_user_identifiers_on_user_id", using: :btree
 
   create_table "users", force: :cascade do |t|
     t.string   "firstname"
     t.string   "surname"
-    t.string   "email",                  default: "",   null: false
+    t.string   "email",                  limit: 80, default: "",   null: false
```

Comment:
not sure where this size limit is coming from. do we want it?

Reply:
It's generally good practice to constrain all string columns where you can. I thought this one was low hanging fruit. I don't think we'll ever see an email address longer than 80 characters?

Reply:
@briri This shouldn't be merged in with these changes. I've updated this branch to revert the change.

Reply:
Correction: Since the email column is indexed on the users table, this raises an error on MySQL dbs when not constrained.

```
ActiveRecord::StatementInvalid: Mysql2::Error: Specified key was too long; max key length is 767 bytes: CREATE UNIQUE INDEX `index_users_on_email` USING btree ON `users` (`email`) 
```

Reverting the previous commit to keep this in place

Reply:
ok. I have no objections to setting character limits in general. No point in allocating unnecessary space if we will never use it.

We had some issues with our templates.links and orgs.links columns in the past where it was changing the between TEXT and VARCHAR and then various size limits simply based on whataver machine had last run a migration. Wanted to make sure it wasn't something similar :)